packages/components/src/DataTable/Header.tsx

```diff
@@ -42,18 +42,26 @@ export function Header<T extends object>({
                   width: header.getSize(),
                   minWidth: header.column.columnDef.minSize,
                   maxWidth: header.column.columnDef.maxSize,
+                  paddingRight: 0,
```

Comment:
I believe this should be applied only if the column is sortable right? If so it's easier to add it into the `sortableColumn` css class.